lms/envs/common.py

```diff
@@ -1620,6 +1620,8 @@
     'survey',
 
     'lms.djangoapps.lms_xblock',
+
+    'openedx.core.djangoapps.content.course_structures',
```

Comment:
I don't think we should gate this PR on a sketch, but we should probably have that conversation now. We can move it later.